entities/mentorship_agreements.ts

```diff
@@ -0,0 +1,8 @@
+interface mentorship_agreement {
+    id: number
+    student_id: number
+    mentorship_start_date: Date
+    mentorship_end_date: Date
```

Comment:
não precisa colocar `mentorship_`, porque já está dentro de `mentoship`.

```suggestion
    start_date: Date
    expected_end_date: Date
```